parameterized/test.py

```diff
@@ -300,3 +300,79 @@ def test_short_repr(input, expected, n=6):
+    ("*a, **kw", param(), []),
+    ("*a, **kw", param(1, foo=42), [("*a", (1,)), ("**kw", {"foo": 42})]),
+    ("foo", param(1), [("foo", 1)]),
+    ("foo, *a", param(1), [("foo", 1)]),
+    ("foo, *a", param(1, 9), [("foo", 1), ("*a", (9,))]),
+    ("foo, *a, **kw", param(1, bar=9), [("foo", 1), ("**kw", {"bar": 9})]),
+    ("x=9", param(), [("x", 9)]),
+    ("x=9", param(1), [("x", 1)]),
+    ("x, y=9, *a, **kw", param(1), [("x", 1), ("y", 9)]),
+    ("x, y=9, *a, **kw", param(1, 2), [("x", 1), ("y", 2)]),
+    ("x, y=9, *a, **kw", param(1, 2, 3), [("x", 1), ("y", 2), ("*a", (3,))]),
+    ("x, y=9, *a, **kw", param(1, y=2), [("x", 1), ("y", 2)]),
+    ("x, y=9, *a, **kw", param(1, z=2), [("x", 1), ("y", 9), ("**kw", {"z": 2})]),
+    ("x, y=9, *a, **kw", param(1, 2, 3, z=3), [("x", 1), ("y", 2), ("*a", (3,)), ("**kw", {"z": 3})]),
+])
+class TestParameterizedClass(TestCase):
+    expect(['TestParameterizedClass_1:test_method_a()',
+            'TestParameterizedClass_1:test_method_b()',
+            'TestParameterizedClass_2:test_method_a()',
+            'TestParameterizedClass_2:test_method_b()',
+            'TestParameterizedClass_3:test_method_a()',
+            'TestParameterizedClass_3:test_method_b()',
+            'TestParameterizedClass_4:test_method_a()',
+            'TestParameterizedClass_4:test_method_b()',
+            'TestParameterizedClass_5:test_method_a()',
+            'TestParameterizedClass_5:test_method_b()',
+            'TestParameterizedClass_6:test_method_a()',
+            'TestParameterizedClass_6:test_method_b()',
+            'TestParameterizedClass_7:test_method_a()',
+            'TestParameterizedClass_7:test_method_b()',
+            'TestParameterizedClass_8:test_method_a()',
+            'TestParameterizedClass_8:test_method_b()',
+            'TestParameterizedClass_9:test_method_a()',
+            'TestParameterizedClass_9:test_method_b()',
+            'TestParameterizedClass_10:test_method_a()',
+            'TestParameterizedClass_10:test_method_b()',
+            'TestParameterizedClass_11:test_method_a()',
+            'TestParameterizedClass_11:test_method_b()',
+            'TestParameterizedClass_12:test_method_a()',
+            'TestParameterizedClass_12:test_method_b()',
+            'TestParameterizedClass_13:test_method_a()',
+            'TestParameterizedClass_13:test_method_b()',
+            'TestParameterizedClass_14:test_method_a()',
+            'TestParameterizedClass_14:test_method_b()',
+            'TestParameterizedClass_15:test_method_a()',
+            'TestParameterizedClass_15:test_method_b()',
+            'TestParameterizedClass_16:test_method_a()',
+            'TestParameterizedClass_16:test_method_b()',
+            'TestParameterizedClass_17:test_method_a()',
+            'TestParameterizedClass_17:test_method_b()',
+            'TestParameterizedClass_18:test_method_a()',
+            'TestParameterizedClass_18:test_method_b()'
+
+            ])
```

Comment:
Sweet! Last minor nitpick here: please update the indent style to follow the style in the rest of the file:

```
expect([
    …
])
```